packages/header-component/README.md

```diff
@@ -14,19 +14,12 @@ Stencil components are just Web Components, so they work in any major framework
 
 ## Getting Started
 
-Currently, due to an issue with lerna and the local dependencies, in order to run this component locally you will need to do the following:
-
-```bash
-yarn remove @debtcollective/dc-dropdown-component
-yarn add @debtcollective/dc-dropdown-component
-```
-
 ```bash
 yarn install
 yarn start
 ```
 
-> In order to have a sense over the community session, you need to have the application running and make sure `community` prop variable match the application, like `http://lvh.me:3000`. Ultimately, you will need to allow CORS on the community app in that sense: Admin > Settings > CORS > add `http://lvh.me:3333` which suppose to be your `host` prop. Check example below.
+> In order to have a sense over the community session, you need to have the application running and make sure `community` prop variable match the application, like `http://lvh.me:3000`. Ultimately, you will need to allow CORS on the community app in that sense: Admin > Settings > CORS > add `http://lvh.me:3333` which suppose to be your `host` prop
```

Comment:
Can we change this so we can point to communit-test.debtcollective.org?
Its unreasonable to have to standup discourse locally to test th header on the gatsby site.
Is this something we can move away from manual configuration and into doppler? (its for env vvariable setting no?)